test/e2e/infra/vagrant/push_flow_aggregator.sh

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env bash
```

Comment:
I feel like instead of all this duplication, we should modify the logic in `./push_antrea.sh` to support uploading multiple docker images to the cluster Nodes

Reply:
I started with that but felt a separate script may help in some scenarios such as extending support such as the automated deployment of elk flow collector. Furthermore, `push_antrea.sh` can be kept simple to be easily extendible with more options in the future.

Reply:
Used one script instead of two as per offline discussion.